rbac/management/group/view.py

```diff
@@ -541,6 +562,18 @@ def principals(self, request, uuid=None):
                 ),
             }
 
+            # Attempt validating and obtaining the "principal type" query
+            # parameter. For now, having "service-account" as the "principal
+            # type" returns a stub.
+            principalType = validate_and_get_key(
+                request.query_params, PRINCIPAL_TYPE_KEY, VALID_PRINCIPAL_TYPE_VALUE, required=False
+            )
+
+            if principalType:
```

Comment:
For these checks, do we want:
```suggestion
            if principalType == "service-account": 
```
so that we still return user-based principals either when it's not supplied _or_ `?principal_type=user`, since we do actually have the user data? Otherwise it might be unexpected if the UI starts getting empty arrays back when pulling existing user data (assuming they use the new query params for everything).

Reply:
Sure. Again sorry, I misunderstood your previous comment. I understood that we had to return the stubbed response regardless of what query param we had specified.